test/controllers/jurisdictions_controller_test.rb

```diff
@@ -99,56 +99,56 @@ def teardown; end
           assert_response :bad_request && next unless user_jur.subtree.include?(jur)
 
           patients = scope == 'all' ? jur.all_patients.where.not(assigned_user: nil) : jur.immediate_patients.where.not(assigned_user: nil)
-          assert_equal patients.distinct.pluck(:assigned_user).sort, JSON.parse(response.body)['assignedUsers']
+          assert_equal patients.distinct.pluck(:assigned_user).sort, JSON.parse(response.body)['assigned_user']
```

Comment:
not sure if this change was accidental, but it's the reason for this test failure: https://github.com/SaraAlert/SaraAlert/pull/450/checks?check_run_id=1321034377#step:14:35

however, I think it would be nice to standardize all variable names with underscores instead of having some of them use underscores and some of them being camelCased. so I would change these to `assigned_users` (this is supposed to be plural because it's a list of existing/possible assigned users). the controller would have be changed here: https://github.com/SaraAlert/SaraAlert/blob/master/app/controllers/jurisdictions_controller.rb#L63

and if we're gonna do it for assigned users, we should do it for jurisdiction paths too: https://github.com/SaraAlert/SaraAlert/blob/master/app/controllers/jurisdictions_controller.rb#L10

the front end stuff will also need to be updated (make sure to replace all occurrences of `jurisdictionPaths` to `jurisdiction_paths` and `assignedUsers` to `assigned_users` as well as their singular form.

I think standardizing all these variables would make it much easier for us to spot any incorrect variable names and it would make sense to include these changes in a refactor like this